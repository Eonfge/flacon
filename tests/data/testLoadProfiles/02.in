[Profiles/AAC]
    Format=AAC
    Name=Aac
    Bitrate=128
    Quality=90
    UseQuality=false
    BitsPerSample=16
    SampleRate=44100

[Profiles/FLAC]
    Format=FLAC
    Name=Flac
    Compression=3
    ReplayGain=Track
    BitsPerSample=16
    SampleRate=44100

[Profiles/MP3]
    Format = MP3
    Name = Mp3
    Bitrate=320
    Preset=vbrStandardFast
    Quality=4
    ReplayGain=Disk
    BitsPerSample=16
    SampleRate=44100

[Profiles/OGG]
    Format = OGG
    Name = Ogg
    MaxBitrate=1
    MinBitrate=1
    NormBitrate=1
    Quality=2
    ReplayGain=Disable
    UseQuality=false
    BitsPerSample=16
    SampleRate=44100

[Profiles/OPUS]
    Name = Opus
    Format = OPUS
    Bitrate=128
    BitrateType=VBR
    BitsPerSample=16
    SampleRate=44100

[Profiles/WV]
    Name = WV
    Format = WV
    Compression=2
    ReplayGain=Track
    BitsPerSample=16
    SampleRate=44100
