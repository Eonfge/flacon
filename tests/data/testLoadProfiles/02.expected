[Profiles/AAC]
    Format=AAC
    Name=Aac
    Bitrate=128
    Quality=90
    UseQuality=false

[Profiles/FLAC]
    Format=FLAC
    Name=Flac
    Compression=3
    ReplayGain=Track

[Profiles/MP3]
    Format = MP3
    Name = Mp3
    Bitrate=320
    Preset=vbrStandardFast
    Quality=4
    ReplayGain=Disk

[Profiles/OGG]
    Format = OGG
    Name = Ogg
    MaxBitrate=1
    MinBitrate=1
    NormBitrate=1
    Quality=2
    ReplayGain=Disable
    UseQuality=false

[Profiles/OPUS]
    Name = Opus
    Format = OPUS
    Bitrate=128
    BitrateType=VBR

[Profiles/WV]
    Name = WV
    Format = WV
    Compression=2
    ReplayGain=Track
